ci/azure-build-release.yml

```diff
@@ -0,0 +1,93 @@
+steps:
+- checkout: self
+  submodules: true
+
+- template: azure-install-rust.yml
+
+- bash: echo "##vso[task.setvariable variable=RUSTC_VERSION;]`rustc --version`"
+  displayName: Set rustc version string for caching
+
+# - bash: |
+#     set -e
+#     curl -Lfo sccache.tar.gz https://github.com/mozilla/sccache/releases/download/0.2.9/sccache-0.2.9-x86_64-apple-darwin.tar.gz
+#     tar -xzf sccache.tar.gz
+#     cp sccache-*/sccache /usr/local/bin/
+#   displayName: Install sccache (OSX)
+#   condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
+
+# - bash: |
+#     set -e
+#     curl -Lfo sccache.tar.gz https://github.com/mozilla/sccache/releases/download/0.2.9/sccache-0.2.9-x86_64-unknown-linux-musl.tar.gz
+#     tar -xzf sccache.tar.gz
+#     sudo cp sccache-*/sccache /usr/local/bin/
+#   displayName: Install sccache (Linux)
+#   condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
+
+# - script: |
+#     curl -Lfo sccache.tar.gz https://github.com/mozilla/sccache/releases/download/0.2.9/sccache-0.2.9-x86_64-pc-windows-msvc.tar.gz
+#     tar -xzf sccache.tar.gz
+#     move sccache-* sccache
+#     echo "##vso[task.prependpath]%CD%\sccache"
+#   displayName: Install sccache (Windows)
+#   condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
+
+- bash: cargo build --release
+  displayName: Cargo build
+
+# Test what we're about to release in release mode itself.
+- bash: cargo test --release --all
+  displayName: Cargo test
+  env:
+    RUST_BACKTRACE: 1
+
+# - script: sccache --show-stats
+#   displayName: post-compile sccache stats
```

Comment:
ditto (I'd like to see these stats :) Unfortunately, it seems that sccache can't handle incremental compilation, making it mostly ineffective.)

Reply:
Turns out the stats are meaningless: sccache only keeps stats for the current session—i.e. until the daemon stops running. But yeah, we didn't see too much of an improvement with it enabled, and it caused issues with the Windows build for Wasmtime. Might be worth looking into with incremental compilation disabled, however.